patch/kernel/archive/rockchip64-6.13/0001-arm64-dts-Add-dts-to-Orange-Pi-5-Max.patch

```diff
@@ -0,0 +1,26 @@
+From b37359e8465822a72d6295a4f66eb4feb011cac6 Mon Sep 17 00:00:00 2001
```

Comment:
I think having this file in dt folder will overwrite the one shipped with the kernel. Means it won't receive future updates from upstream. This would also probably cause breakage on other boards that build their device tree on top of the opi5 base one.
The `dt` folder should only be used for complete (no includes/dependencies) device trees for individual boards.
Same goes for 6.13.

Reply:
This file is not in `dt` folder, it is patch which is only adding one line in `dts/rockchip/Makefile`:
`+dtb-$(CONFIG_ARCH_ROCKCHIP) += rk3588-orangepi-5-max.dtb`
Or I didn't get the point.